Dockerfile

```diff
@@ -15,11 +15,15 @@ RUN dnf -y update && \
         wget \
         git \
         python36 \
-        java-1.8.0-openjdk-devel \
+        java-11-openjdk-devel \
         maven \
         cargo \
     && dnf clean all
 
+# ensure we're actually using java 11
+ENV JAVA_HOME=/usr/lib/jvm/java-11-openjdk
+RUN alternatives --set java /usr/lib/jvm/java-11-openjdk/bin/java
+
```

Comment:
setting JAVA_HOME and alternatives is unnecessary here, there's only one jdk installed:
```suggestion
```

Reply:
java-1.8.0-openjdk-devel seems to be uninstalled by default, which is why I put this in. Let me try removing it.

Reply:
it's not installed by default, it's being pulled in by maven

Reply:
I see. In any case, yes, it seems like this command *is* necessary (although the invocation of it here was slightly incorrect) since java 1.8.0 gets selected as default otherwise.